Foods.yaml

```
@@ -6,5 +6,5 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["Pizza"]
+  - ["Pizza", "Rat", "Sewage water"]
```

It was pointed out to me yesterday that we should try to stick to `snake_case` for `yaml` files in the NUbots codebase
```suggestion
  - ["pizza", "rat", "sewage_water"]
```

---

Do we have an agreed upon set of standards for these yet? Otherwise, that would be quite useful.